rtos/TARGET_CORTEX/mbed_rtx_idle.cpp

```diff
@@ -24,7 +24,11 @@
 #include "platform/mbed_sleep.h"
 #include "TimerEvent.h"
 #include "lp_ticker_api.h"
+#if defined(__CORTEX_A9)
+#include "rtx_core_ca.h"
+#else
 #include "rtx_core_cm.h"
+#endif
```

Comment:
Why do you need to access this internal RTX header?
A user should never need to rely on RTX implementation details like this.

Reply:
I don't actually think it's needed.

Reply:
It seems that it implemented by this commit (https://github.com/ARMmbed/mbed-os/commit/4b354f80cfbd819a951b9ac93ea7c781f3d5a77b#diff-415fd494349b5414064e0cc0d9f0d596).
In cortex-A, the build error occurs by including "rtx_core_cm.h". Therefore I implemented to include the header file for Cortex-A("rtx_core_ca.h").

Reply:
Sure rtx_core_cm.h contains the core abstraction for Cortex-M, so this cannot be used for Cortex-A. I strongly recommend not to rely on internal stuff. Actually the include (formerly core_cm.h) was introduced in #e44d94f. Can you point out what you need that's defined in this header?

Reply:
It's not needed and can be deleted. I'll submit an PR to do it.